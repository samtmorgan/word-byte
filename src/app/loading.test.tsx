import React from 'react';
import Loading from './loading';

describe('loading page', () => {
  it('should return a loading component', () => {
    const result = Loading();
    expect(result).toEqual(<div>Loading...</div>);
  });
});
