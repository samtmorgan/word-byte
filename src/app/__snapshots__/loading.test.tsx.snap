// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading page should return the loading component 1`] = `
<div>
  <div>
    mock loading component
  </div>
</div>
`;
